personalidade/style.css

```diff
@@ -0,0 +1,212 @@
+
+body{
+    font-family: 'Montserrat', sans-serif; 
+    margin: 0;
+    background-color: #E5E5E5;
+}
+
+.cabecalho__nav{
+    width: 100%; 
+    background-color: #674582
+}
+.nav__content{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+.logo{
+    width: 250px;
+}
+.logo__img{
+    width:100%;
+}
+
+.cabecalho__home a{
+    text-decoration: none;
+    margin:20px;
+    color: #fff;
+}
+.redes__sociais{
+    width: 150px;
+}
+
+.header{
+    display: flex;
+    align-items: flex-end;
+    background-image: url(../personalidade/carolina-maria/img/banner-carolina.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 400px;
+}
+.titulo__banner{
+    text-align: center;
+    color:#fff;
+    width: 100%;
+    text-transform: uppercase;
+}
+/* inicio do conteúdo */
+
+.titulo__citacao{
+    align-items: center;
+    display: flex;
+    color:#63458A; 
+    justify-content: center;
+}
+
+/* .container{
+    display: flex;
+    font-family: Montserrat;
+    flex-direction: row;
+    justify-content: center;
+    color:#63458A;
+} */
+.personalidade{
+    display: flex;
+    margin-top: 50px;
+    color:#63458A; 
+}
+.img__carolinadejesus{
+    width: 30%;
+}
+
+.perfil__personalidade{
+    width:70%;
+}
+
+/* .perfil__personalidade h4{
+    display: flex;
+    flex-direction: row;
+}
+
+.perfil__personalidade p{
+    width: 50%;
+    justify-content: center; */
+/* } */
+ 
+.img__autora{
+    width: 201px;
+    justify-content: center;
+}
+/* borda e citacao*/
+
+.citacao{
+    border: 0.5px solid #63458A;
+    display: flex;
+    justify-content: center;
+    color:#63458A;
+    margin: 0 auto;
+    width: 660px;
+    margin-top: 50px;
+}
+
+.historia__personalidade{
+    margin: 0 auto;
+    width: 660px;
+    margin-top: 50px;
+    color:#63458A;
+}
+
+.foto__carolina{
+    width: 90%;
+    align-items: center;
+}
+/* inicio perfil da autora */
+.autora__perfil{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #F6B0DC;
+    color:#63458A;  
+}
+.foto__autora{
+    width: 201px; 
+    height:201px; 
+}
+
+.autora__perfil h6{
+    width: 400px;
+    margin-left: 20px;
```

Comment:
tenta alinhar com um conatiner pra depois, quando for fazer o responsivo,  não ter problemas com alinhamento